pkg/inputs/snmp/metrics/device_metrics.go

```diff
@@ -271,26 +271,6 @@ func (dm *DeviceMetrics) pollFromConfig(ctx context.Context, server *gosnmp.GoSN
 		dst.CustomStr["profile_message"] = kt.DefaultProfileMessage
 	}
 
```

Comment:
Don't need this code block any more because the measurements come in via a separate thread.